packages/driver/index.d.ts

```diff
@@ -2,4 +2,5 @@
 /// <reference path="../ts/index.d.ts" />
 export const $Cypress: Cypress.Cypress
 
-export default $Cypress
+export const $: typeof JQuery
```

Comment:
Why was this change necessary?
It seems not connected to the PR. It might be a dependency of the work.

Usually, this is supposed to be part of the jQuery types inside of your ``tsconfig,json`

Reply:
Many places in the codebase do `import { $} from '@packages/driver'`. Until now they were all in JS files, so no-one noticed but the types were wrong. I added some TS files in this PR, revealing the wrong typings. So I fixed the types.